tests/MongoFramework.Tests/MongoFramework.Tests.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <AssemblyName>MongoFramework.Tests</AssemblyName>
     <RootNamespace>MongoFramework.Tests</RootNamespace>
-    <TargetFrameworks>net461;netcoreapp2.1;netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>net461;netcoreapp2.1;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

Comment:
Let's remove .NET Core 2.1 from here as it is no longer supported.

Reply:
Removed